### ISSUE TITLE
(Python) AssembliesApi: all GET calls return None

### ISSUE THREAD

**Issue by francesco-romano** [onshape-public/onshape-clients#18]
All the (GET) calls in the `AssembliesApi` [1] return None.
But I am expecting the calls to return some content, as I can see the `json` content in the API explorer

**Comment by ethan92429** [comment]
Yup - still working on typing this part of the API. Will let you know when this is done and thanks for posting!

**Comment by francesco-romano** [comment]
Thanks for your reply.
Do you have an indicative ETA?

**Comment by ethan92429** [comment]
Hopefully sometime later this month.

**Comment by pmdhazy** [comment]
Is this the same problem for part_studios_api.export_stl1 returning None as well?

I found a workaround, pass _preload_content=False to the call.

**Comment by ethan92429** [comment]
Yes - in general you will get this when the response content isn't specified in the openapi spec. Passing _preload_content=False is the right workaround.

**Comment by rsinnet** [comment]
I think this issue can be closed. The assemblies API seems to work.
